tests/director/tests/main_test.go

```diff
@@ -29,14 +31,20 @@ type DirectorConfig struct {
 	DefaultNormalizationPrefix     string `envconfig:"default=mp-"`
 	GatewayOauth                   string
 	DirectorExternalCertSecuredURL string
-	SkipSSLValidation              bool `envconfig:"default=false"`
+	SkipSSLValidation              bool   `envconfig:"default=false"`
+	ConsumerID                     string `envconfig:"APP_INFO_CERT_CONSUMER_ID"`
 	CertLoaderConfig               certloader.Config
 	certprovider.ExternalCertProviderConfig
-	SelfRegLabelKey              string `envconfig:"APP_SELF_REGISTER_LABEL_KEY"`
-	SelfRegDistinguishLabelKey   string
-	SelfRegDistinguishLabelValue string
-	SelfRegRegion                string
-	ConsumerID                   string `envconfig:"APP_INFO_CERT_CONSUMER_ID"`
+	SubscriptionConfig                  subscription.Config
+	TestProviderSubaccountID            string
+	TestConsumerSubaccountID            string
+	TestConsumerTenantID                string
+	ExternalServicesMockBaseURL         string
+	TokenPath                           string
+	SubscriptionProviderAppNameLabelKey string
+	SubscriptionProviderAppNameValue    string
+	ConsumerSubaccountLabelKey          string
+	SubscriptionLabelKey                string
```

Comment:
maybe missing envconfig? in the yaml files they're prepended with an `APP_` and I think this will match `SUBSCRIPTION_LABEL_KEY`. same applies to other new fields.

Reply:
I'm not sure I understand the comment but it should be okay. I double checked them and in the director-test.yaml e.g. there is APP_SUBSCRIPTION_LABEL_KEY - https://github.com/kyma-incubator/compass/blob/ff84c1cf1b225cad9cfbf282543fa6cf7429cace/chart/compass/templates/tests/director/director-test.yaml#L158
The `APP_` prefix is just an configuration of the env library - https://github.com/kyma-incubator/compass/blob/ff84c1cf1b225cad9cfbf282543fa6cf7429cace/tests/pkg/config/utils.go#L11